bigquery/snippets/snippet_test.go

```diff
@@ -33,6 +34,26 @@ func init() {
 	}
 }
 
+// uniqueBQName returns a more unique name for a BigQuery resource.
+func uniqueBQName(prefix string) string {
+	t := time.Now()
+	return fmt.Sprintf("%s_%d_%d", sanitize(prefix, '_'), t.Unix(), t.Nanosecond())
```

Comment:
I think we can drop the nanosecond. I know bucket name lengths can be limited. Not sure about BQ.

Reply:
bucket names are bounded at 63 chars except for the dns-style name cases, so I added a check that truncates.

Reply:
BQ references can be up to 1k chars, so not worrying about that.